pages/createForm/index.js

```diff
@@ -0,0 +1,28 @@
+import ActivityForm from "@/components/ActivityForm";
+import Link from "next/link";
+import styled from "styled-components";
+
+const StyledAddLink = styled(Link)`
+  background-color: orange;
+  text-decoration: none;
+  color: black;
+  border: 1px solid black;
+  padding: 5px;
+  border-radius: 5px;
+`;
+
+const StyledFieldset = styled.fieldset`
+  margin: 20px 30px 50px;
+`;
+
+export default function CreateForm({ onAddActivity }) {
+  return (
+    <>
+      <StyledAddLink href={"/"}>Go back</StyledAddLink>
+      <StyledFieldset>
```

Comment:
A 'fieldset' element is used for grouping controls INSIDE a form. If you want exact this styling move the fieldset inside the form.